examples/crhmc_sampling/common.hpp

```diff
@@ -0,0 +1,61 @@
+#ifndef COMMON_HPP
```

Comment:
Can you give a more descriptive name to the file?

Reply:
done